test/testnet/localnet/Makefile

```diff
@@ -0,0 +1,47 @@
+.PHONY: deploy stop update-prereq build-tendermint status
+
+ACTIVATE_VENV := source venv/bin/activate
+TESTNET ?= tik
+
+build-tendermint:
+	@echo "Building Tendermint binary for Linux..."; \
+	cd ${GOPATH}/src/github.com/tendermint/tendermint; \
+	make build-linux; \
```

Comment:
`make -C ${GOPATH}/src/github.com/tendermint/tendermint build-linux` can do this in one line for you and the "done" line is unneccessary. (Makefile makes a similar report at the end if I recall.)
No need for multi-lines this way.

Reply:
Oh, sorry, one more note: you don't call `make` in a Makefile, you call `$(MAKE)`. The command looks like this:
`$(MAKE) -C ${GOPATH}/src/github.com/tendermint/tendermint build-linux`

Unless you want to include that file, in which case you can just call the `build-linux` target directly. :)